src/App.js

```diff
@@ -1,27 +1,13 @@
-import logo from './logo.svg';
 import './App.css';
+import HomePage from './pages/home-page/HomePage';
+import Header from './componentes/shared/header/Header';
```

Comment:
Add entry point index.js 
```suggestion
import Header from './componentes/;
```